o1vm/run-code.sh

```diff
@@ -10,6 +10,10 @@ set -u
 
 source $FILENAME
 
-./run-op-program.sh
-./run-cannon.sh
+if [ -d "op-program-db-for-latest-l2-block" ] && [ -f "env-for-latest-l2-block.sh" ] && [ -f "snapshot-state-3000000.json" ] && [ -f "meta.json" ]; then
```

Comment:
I'm a bit against this. Having this condition doesn't mean we don't want to get the latest data. I would keep this file simple.

Reply:
Looking at the command in `run-vm.sh`:
```
cargo run --bin ${BINARY_FLAVOR} \
    --all-features \
    --release \
    -p o1vm -- \
      --pprof.cpu \
      --info-at "${INFO_AT:-%10000000}" \
      --snapshot-state-at "${SNAPSHOT_STATE_AT:-%10000000}" \
      --proof-at never \
      --stop-at "${STOP_AT:-never}" \
      --input "${ZKVM_STATE_FILENAME:-./state.json}" \
      -- \
      ./ethereum-optimism/op-program/bin/op-program \
      --log.level DEBUG \
      --l1 "${L1_RPC}" \
      --l2 "${L2_RPC}" \
      --network sepolia \
      --datadir "${OP_PROGRAM_DATA_DIR}" \
      --l1.head "${L1_HEAD}" \
      --l2.head "${L2_HEAD}" \
      --l2.outputroot "${STARTING_OUTPUT_ROOT}" \
      --l2.claim "${L2_CLAIM}" \
      --l2.blocknumber "${L2_BLOCK_NUMBER}" \
      --server
```
`OP_PROGRAM_DATA_DIR` is `op-program-db-for-latest-l2-block`
`FILENAME` is `env-for-latest-l2-block.sh`
`ZKVM_STATE_FILENAME` is `snapshot-state-3000000.json`


`op-program-db-for-latest-l2-block` has